modules/math/shared/src/main/scala/gsp/math/Wavelength.scala

```diff
@@ -11,7 +11,7 @@ import monocle.Prism
 /**
  * Exact wavelengths represented as unsigned integral picometers in the range [0 .. Int.MaxValue]
  * which means the largest representable wavelength is 2.147483647 mm.
- * @param toPicometers This wavelength in integral picometers (10^-12 of a meter).
+ * @param toPicometers This wavelength in integral picometers (10^-12^ of a meter).
```

Comment:
My intent was `10^-12` ... does `^foo^` make a superscript in Scaladoc?

Reply:
Yes.  You could argue though that mostly the documentation is read in the source code and so this change is more confusing. I can put it back.

Reply:
Ok. Either way. I tried `10⁻¹²` but it's unreadable in my terminal font.